deployalo.com.subdomain.json

```
@@ -0,0 +1,22 @@
+{
+  "providerId": "deployalo.com",
+  "providerName": "Deployalo",
+  "serviceId": "subdomain",
+  "serviceName": "Deployalo Subdomain",
+  "version": 1,
+  "logoUrl": "https://deployalo.com/deployalo-icon.png",
+  "description": "Point a subdomain to your Deployalo deployment",
+  "variableDescription": "The subdomain of your Deployalo deployment (e.g., sunny-wind.deployalo.com)",
+  "syncPubKeyDomain": "deployalo.com",
+  "syncRedirectDomain": "deployalo.com",
+  "warnPhishing": true,
+  "hostRequired": true,
+  "records": [
+    {
+      "type": "CNAME",
+      "host": "%host%",
```

Please see the linter warning. 
```suggestion
      "host": "@",
```